assets/icons/musescore-icon-round.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000">
+    <linearGradient id="a" gradientUnits="userSpaceOnUse" x1="13.353" y1="503.002" x2="971.535" y2="503.002" gradientTransform="rotate(44.684 491.944 503.442)">
```

Comment:
None of the colours in the logo quite match the blue used on the website. The website uses `#1f74bd` (or `rgb(31, 116, 189)`). Please make sure this exact colour is used somewhere in the gradient.

Reply:
Currently its implicitly in the middle of the gradient, but I'll add it explicitly now.